src/strategies/information.rs

```diff
@@ -900,7 +905,7 @@ impl PlayerStrategy for InformationPlayerStrategy {
         let public_useless_indices = self.find_useless_cards(view, &self.get_my_public_info());
         let useless_indices = self.find_useless_cards(view, &private_info);
 
-        if view.board.discard_size() <= discard_threshold {
+        if view.board.discard_size() < soft_discard_threshold {
```

Comment:
thanks for the change!

is the 5 player result change just noise?  If not, it would be easy to recover the old behavior by changing this to `<=` right? (and changing the < 5 player case to `discard_threshold - 5`)

Reply:
Yeah, sorry I didn't notice that! I'll fix it and then rebase everything.